### PR TITLE
[1603] Fix registration path

### PR DESCRIPTION
**Pull request by blazarecki**

### CODE REVIEW

File: src/Controller/EnMarche/MembershipController.php [hunk 1]
```diff
@@ -28,19 +29,28 @@ class MembershipController extends Controller
...
      * This action enables a guest user to adhere to the community.
      *
-     * @Route("/adhesion/{uuid}", name="app_register_fully")
+     * @Route("/adhesion", name="app_register_fully")
      * @Method("GET|POST")
+     *
+     * @Security("is_granted('ROLE_USER')")
      */
-    public function finaliseRegistrationAction(Request $request, Adherent $adherent): Response
+    public function finaliseRegistrationAction(Request $request): Response
```
Comment (HeahDude) [inline src/Controller/EnMarche/MembershipController.php]: not related to your PR, but typo: finalize (we use American English).

File: src/Entity/Adherent.php [hunk 2]
```diff
@@ -516,6 +529,7 @@ public function updateMembership(MembershipRequest $membership, PostAddress $pos
         $this->phone = $membership->getPhone();
         $this->comEmail = $membership->comEmail;
         $this->comMobile = $membership->comMobile;
+        $this->adherent = true;
```
Comment (HeahDude) [inline src/Entity/Adherent.php]: 🤔 So editing a profile is enough to be adherent?

File: templates/base.html.twig [hunk 3]
```diff
@@ -6,7 +6,7 @@
 {% set default_sharer_width = 1905 %}
 {% set default_sharer_height = 1000 %}
 
-{% set user_is_adherent = is_granted('ROLE_ADHERENT') %}
+{% set user_is_user = is_granted('ROLE_USER') %}
```
Comment (HeahDude) [inline templates/base.html.twig]: `user_is_adherent`? the current naming looks weird.

**Comment by HeahDude** [comment]
I'm not a fan of this new `adherent` profile property. What's the process now to be one or to stop being one?

File: templates/base.html.twig (hunk 3 above)
Comment (blazarecki) [reply templates/base.html.twig]: Now you can be registered in en-marche without being an adherent. The naming is weird.

File: src/Entity/Adherent.php (hunk 2 above)
Comment (blazarecki) [reply src/Entity/Adherent.php]: No, finalize your registration in en-marche is enough

File: src/Controller/EnMarche/MembershipController.php (hunk 1 above)
Comment (blazarecki) [reply src/Controller/EnMarche/MembershipController.php]: I'll rename it.

**Comment by blazarecki** [comment]
You register in auth.en-marche.fr from en-marche.fr -> you became a `User`.

File: src/Entity/Adherent.php (hunk 2 above)
Comment (HeahDude) [reply src/Entity/Adherent.php]: So this should be removed, this makes it true for any edition (update) of the profile.

**Pull request merged**